src/isar_exr/api/energy_robotics_api.py

```diff
@@ -485,8 +551,13 @@ def start_mission_execution(self, mission_definition_id: str, robot_id: str) ->
             response_dict: dict[str, Any] = self.client.query(
                 dsl_gql(start_mission_execution_mutation), params
             )
-        except Exception as e:
-            raise RobotException from e
+        except Exception:
+            message: str = "Could not start mission execution"
+            self.logger.error(message)
+            raise RobotException(
```

Comment:
Why not raise `RobotInfeasibleMissionException` and just pass the error description?